src/GitVersionCore/Extensions/ObjectExtensions.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace GitVersion.Extensions
+{
+    public static class ObjectExtensions
+    {
+        public sealed class ReflectionIgnoreAttribute : Attribute
```

Comment:
Perhaps internal makes more sense
```suggestion
        internal sealed class ReflectionIgnoreAttribute : Attribute
```

Reply:
Sure.